docs/topics/jvm/jvm-spring-boot-restful.md

```diff
@@ -0,0 +1,177 @@
+[//]: # (title: Create a RESTful web service with Spring Boot)
+
+This tutorial walks you through the process of creating a simple application with Spring Boot.
+
+You will create an application with the HTTP endpoint that returns a data objects list in the JSON format.  
+
+To get started, first download and install the latest version of [IntelliJ IDEA](http://www.jetbrains.com/idea/download/index.html).
+
+## Bootstrap the project
+
+To generate a new project, use the Spring Initializr:
+
+> You can also create a new project using [IntelliJ IDEA with the Spring Boot plugin](https://www.jetbrains.com/help/idea/spring-boot.html) 
+>
+{type="note"}
+
+1. Open the [Spring Initializr](https://start.spring.io/#!type=gradle-project&language=kotlin&platformVersion=2.4.2.RELEASE&packaging=jar&jvmVersion=11&groupId=com.example&artifactId=demo&name=demo&description=Demo%20project%20for%20Spring%20Boot&packageName=demo&dependencies=web,data-jdbc,h2). The link from the tutorial opens the window with the predefined settings of the new project. 
+  This project uses **Gradle** as a build tool, **Kotlin** as a language of choice, and the following dependencies: **Spring Web**, **Spring Data JDBC**, and **H2 Database**:
+
+    ![Create a new project with Spring Initializr](spring-boot-create-project-with-initializr.png){width=800}
+
+2. Click **GENERATE** at the bottom of the screen. Spring Initializr will generate the project with the specified settings. The download starts automatically.
+
+3. Unpack the **.zip** file and open it in the IntelliJ IDEA.
+  
+   The project has the following structure: 
+   ![The Spring Boot project structure](spring-boot-project-structure.png)
+   
+    There are packages and classes under the `main/kotlin` folder that belong to the application. The entry point to the application is the `main()` method of the `DemoApplication.kt` file.
+
+## Explore the project build file
+
+Open the `build.gradle.kts` file:
+
+```kotlin
```

Comment:
Not sure about the value of the whole build script here. What do you want to say with it? Can we avoid listing third-party code? It will be hard to support it: there are a lot of versions (including Kotlin 1.4.21 which is outdated in scope of our docs), and any other changes can happen too.